modules/auxiliary/scanner/sap/sap_soap_bapi_user_create1.rb

```diff
@@ -81,6 +81,7 @@ def run_host(ip)
         'headers' => {
           'SOAPAction' => 'urn:sap-com:document:sap:rfc:functions',
         },
+        'encode_params' => false,
```

Comment:
```sap-client``` is user-supplied so I guess it's possible to get URI-encoded... I'm only trying to restore the original behavior that I know for sure should work because they were originally written that way.